config.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
      * you do not plan on using the SDK.  It is, however, strongly recommended
      * you use the SDK.
      */
-    twofactorUrl: 'https://mfa.bandwidth.com/app/two-factor',
+    twofactorUrl: (accId, endpoint) => {return 'https://mfa.bandwidth.com/api/v1/accounts/' + accId + '/code/' + endpoint},
```

Comment:
more in the style of a usual javascript/typescript app, especially customer facing, would be to name this var accountId